docker/sawtooth-int-block-info-tp

```diff
@@ -21,7 +21,7 @@
 #   This image should be built using `build_all installed`.
 #
 # Run:
-#   $ docker run sawtooth-block-info-tp
+#   $ docker run sawtooth-int-block-info-tp
```

Comment:
This is not a typo, the Dockerfile is `sawtooth-int-block-info-tp` but the image created by build_all is called `sawtooth-block-info-tp`.